### PR TITLE
keep things dry

### PR DESCRIPTION
**Pull request by michaellzc**
keep things DRY

### CODE REVIEW

File: charts/sourcegraph/templates/indexed-search/indexed-search.StatefulSet.yaml [hunk 1]
```diff
@@ -55,14 +55,8 @@ spec:
...
         {{- if .Values.otelCollector.enabled }}
-        # OTEL_AGENT_HOST must be defined before OTEL_EXPORTER_OTLP_ENDPOINT to substitute the node IP on which the DaemonSet pod instance runs in the latter variable
-        - name: OTEL_AGENT_HOST
-          valueFrom:
-            fieldRef:
-              fieldPath: status.hostIP
-        - name: OTEL_EXPORTER_OTLP_ENDPOINT
-          value: http://$(OTEL_AGENT_HOST):4317
         - name: OPENTELEMETRY_DISABLED
           value: "false"
```
Comment (michaellzc) [inline charts/sourcegraph/templates/indexed-search/indexed-search.StatefulSet.yaml:61]: I guess zoekt has otel disabled by default?

Comment (bobheadxi) [reply charts/sourcegraph/templates/indexed-search/indexed-search.StatefulSet.yaml:61]: disabled = false => enabled :)

Comment (michaellzc) [reply charts/sourcegraph/templates/indexed-search/indexed-search.StatefulSet.yaml:61]: lol I just noticed it's `DISABLED` instead of `ENABLED`

**Review by sanderginn — CHANGES_REQUESTED** [review]
Neat, thank you 🙂 
The source branch was just updated to name the values `openTelemetry` instead of `otelCollector`, so just need to reflect that here

**Review by sanderginn — APPROVED** [review]

**Pull request merged**